torch/csrc/api/include/torch/special.h

```diff
@@ -452,6 +452,116 @@ inline Tensor& zeta_out(
   return torch::special_zeta_out(result, self, other);
 }
 
+/// Computes betaln
```

Comment:
I would suggest removing the scalar overloads here as well to simplify significantly the logic here.